src/tito/tagger/main.py

```diff
@@ -478,9 +478,16 @@ def _update_package_metadata(self, new_version):
         run_command("git add %s" % os.path.join(self.full_project_dir,
             self.spec_file_name))
 
-        run_command('git commit -m "Automatic commit of package ' +
-                '[%s] %s [%s]."' % (self.project_name, self.release_type(),
-                    new_version_w_suffix))
+        fmt = ('Automatic commit of package '
+               '[%(name)s] %(release_type)s [%(version)s].')
+        if self.config.has_option(BUILDCONFIG_SECTION, "commit_message_format"):
+            fmt = self.config.get(BUILDCONFIG_SECTION, "commit_message_format")
```

Comment:
If you are reading this config, then you should escape that value. What if the message in config contains quotes?